jsonnet/manifests/noobaa-prometheus-rules.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
@umangachapagain for my understanding, why do we need the manifest committed? Can't we bring it from the dependent repo?

Reply:
Ideally, noobaa-mixins should not have this manifest committed. It's scope should be limited to generating the  [alerting rule](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/#defining-alerting-rules) file. (which is why I was thinking of a cleanup PR there)

Since it is the use-case of the noobaa-operator that this alerting file needs to be deployed in a K8s/Openshift environment, it should be responsible for handling creation of this manifest. (with all the customized metadata)

That being said, we could choose to generate this file at build time and copy it into the container (without a commit it here). But, from what I have seen, all other projects commit this manifest when something changes. (makes it easier to track what went into the container and user can always directly use this for manual deploy)

Reply:
Hi @umangachapagain I agree - it makes sense, just like when generating the CRD's using operator-sdk generate...

Reply:
That is a good example. Never thought of it :P

Reply:
There is a bug in this CRD. It won't apply.
I've got a PR in NooBaa Mixins that I'm waiting to merge which fixes that.
It would be great if you could update the package.lock with the newer master and generate this CRD once again with the new commit.
I will update here once I merge the PR.

Reply:
Can you provide a link to that PR?

Reply:
https://github.com/noobaa/noobaa-mixins/pull/11
I've merged the PR.
Please update the package.lock and generate the CRD with the new changes.
Thank you.

Reply:
done.